app/persistence/fabric/CRUDService.ts

```diff
@@ -103,17 +104,36 @@ export class CRUDService {
 		let sqlTxList = ` select t.creator_msp_id,t.txhash,t.type,t.chaincodename,t.createdt,channel.name as channelName from transactions as t
        inner join channel on t.channel_genesis_hash=channel.channel_genesis_hash and t.network_name = channel.network_name where  t.blockid >= $1 and t.id >= $2 and
 							t.channel_genesis_hash = $3 and t.network_name = $4 and t.createdt between $5 and $6 `;
-		const values = [blockNum, txid, channel_genesis_hash, network_name, from, to, page, size];
+		const values = [
+			blockNum,
+			txid,
+			channel_genesis_hash,
+			network_name,
+			from,
+			to,
+			page,
+			size
+		];
 		if (page == 1) {
 			let sqlTxCount: string;
-			const filterValues = [blockNum, txid, channel_genesis_hash, network_name, from, to];
+			const filterValues = [
```

Comment:
Please take care of formatting, as it shouldnt disturb the oriniginal formatting of the file.